esmvaltool/diag_scripts/climate_metrics/ecs.py

```diff
@@ -57,153 +56,128 @@ def plot_ecs_regression(cfg, dataset_name, data, variables, regression_stats):
            r'$\alpha$ = {:.2f}, '.format(-regression_stats.slope) + \
            'F = {:.2f}, '.format(regression_stats.intercept) + \
            'ECS = {:.2f}'.format(ecs)
-    e.plot.scatterplot(
-        [data[0], x_reg],
-        [data[1], y_reg],
-        filepath,
-        plot_kwargs=[{'linestyle': 'none',
-                      'markeredgecolor': 'b',
-                      'markerfacecolor': 'none',
-                      'marker': 's'},
-                     {'color': 'k',
-                      'linestyle': '-'}],
-        save_kwargs={
-            'bbox_inches': 'tight',
-            'orientation': 'landscape'},
-        axes_functions={
-            'set_title': dataset_name,
-            'set_xlabel': 'tas / ' + variables.units('tas'),
-            'set_ylabel': 'rtmt / ' + variables.units('rtmt'),
-            'set_xlim': [0.0, 7.0],
-            'set_ylim': [-2.0, 10.0],
-            'text': {'args': [0.05, 0.9, text],
-                     'kwargs': {'transform': 'transAxes'}}})
+    plot.scatterplot([tas_cube.data, x_reg], [rtmt_cube.data, y_reg],
```

Comment:
Add comma's after the last item in lists, dicts, tuples, function arguments when they span multple lines, see also [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas). Doing this will also make `yapf` format them correctly, solving your Codacy issue.